### DISCUSSION TITLE
[FR] Add a static xcframework to tags.

### DISCUSSION THREAD

**Discussion by jesus-mg-ios** [airbnb/lottie-ios#2486] [General]
## Which Version of Lottie are you using?

Lottie 4.5.0

## Expected Behavior 
Have a way to download precompiled static xcframework. 

## Actual Behavior
The xcframework right now is dynamic, you can check it with otool.

**Comment by marinofelipe** [comment]
Are there any updates on this? I'd love if a static XCFramework was provided so that it doesn't impact our app launch times between other benefits for our setup 🙏

**Comment by Loupehope** [comment]
Now the release artifacts include a static framework - [Lottie-Static.xcframework.zip](https://github.com/airbnb/lottie-ios/releases/tag/4.5.2)
